contracts/proto/MockClient.sol

```diff
@@ -5,734 +5,732 @@ import "./GoogleProtobufAny.sol";
 import "./Client.sol";
 
 library IbcLightclientsMockV1ClientState {
+    //struct definition
+    struct Data {
+        Height.Data latestHeight;
+    }
+
+    // Decoder section
+
+    /**
+     * @dev The main decoder for memory
+     * @param bs The bytes array to be decoded
+     * @return The decoded struct
+     */
+    function decode(bytes memory bs) internal pure returns (Data memory) {
+        (Data memory x, ) = _decode(32, bs, bs.length);
+        return x;
+    }
+
+    /**
+     * @dev The main decoder for storage
+     * @param self The in-storage struct
+     * @param bs The bytes array to be decoded
+     */
+    function decode(Data storage self, bytes memory bs) internal {
+        (Data memory x, ) = _decode(32, bs, bs.length);
+        store(x, self);
+    }
+
+    // inner decoder
+
+    /**
+     * @dev The decoder for internal usage
+     * @param p The offset of bytes array to start decode
+     * @param bs The bytes array to be decoded
+     * @param sz The number of bytes expected
+     * @return The decoded struct
+     * @return The number of bytes decoded
+     */
+    function _decode(
+        uint256 p,
+        bytes memory bs,
+        uint256 sz
+    ) internal pure returns (Data memory, uint) {
```

Comment:
Is this file just formatted or are there other changes?

Reply:
Yes, can confirm that it was just formatted.

Reply:
this file formatted automatically by the editor